### ISSUE TITLE
Specify known good versions of JDK and Leiningen in documentation

### ISSUE THREAD

**Issue by phlummox** [LuxLang/lux#54]
It seems it's a common stumbling block for people to be unable to build even a "Hello world" program in Lux due to their using versions of Leiningen and the JDK that Lux doesn't work with - e.g. #41.

Rather than just closing those issues, could I suggest that it might, perhaps, be useful to add to both the [README][readme-link] and to the [Lux book][lux-book] (assuming the URL I have for it is up to date - perhaps this has already been added) a new section called "Prerequisites" which details the versions of JDK and Leiningen which Lux is known to work with?

[readme-link]: https://github.com/LuxLang/lux/blob/cf9e65352cb477982754c98fafe0a3a98a42670d/README.md

[lux-book]: https://luxlang.gitbooks.io/the-lux-programming-language/

**Comment by eduardoejp** [comment]
Hello, @phlummox !

I don't think I can give you a satisfactory answer for this, but at least I'd like to let you know where things stand.

I have focused (for a while... >_>) on finishing version 0.6 of the language.
Version 0.6 works with the latest Leiningen, and on Java 13.
Which is to say, I addressed the issue that you mentioned... but only for v0.6.

That is because I haven't worked on keeping v0.5 (which is the version used in the book) up to date, since I feel there's not much point in doing so since v0.6 is what I need to be focusing on.

I realize that's not a good answer for someone who just wants to try out Lux and only has access to v0.5 (and the only available book only covers v0.5).

For what is worth, I expect to have a release-ready v0.6 JVM compiler by the end of June or early July.
I could also update parts of the book on demand for people who want to play around with it.

But right now I don't want to spend time updating v0.5 or back-porting v0.6 changes into it, as I feel it would be a distraction.

I apologize for the current state of affairs. It's mostly a consequence of how long it has taken me to develop v0.6, but hopefully it will all be worth it soon.
